### PR TITLE
Not to merge yet there are still issues

### PR DESCRIPTION
**Pull request by someknave**
Okay so I have been getting nowhere with this, scoping is killing me and all my best efforts are not getting there.
The  getNameTokens() function needs to be outside the params variable in order to work at all. but when outisde, no matter how I call it I can't get it to properly reference the world.indexOf call correctly. you can see that I have bypassed this with a dummy conditional for testing purposes, the rest of this function seems to work as specified.

Then the tokens. tokenNames seems to be an object that correctly stores {agent: 'player', ent: 'player', fowns: 'his', owns: 'his'}. {{tokenNames.agent}} correctly prints "player" on the finished page. but the separate lines like 'agent': tokenNames.agent, cause the page to error. I need a little help here.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1658,7 +1658,18 @@
 			arr[j] = temp;
 		}
 	}
-
+	
+	function getNameTokens() {
+		var tokens;
+	//	if(this.world.indexOf('9') == 1 ||this.world.indexOf('9') == 0) {
```
Comment (Arachnid) [inline index.html]: 'this' isn't defined here. Instead, take 'world' as an argument to the function, and pass it below, in load().

File: index.html [hunk 2]
```diff
@@ -1703,12 +1713,18 @@
 			'mapnum': {},
 			'order': priority_match([modules['I']['order'], modules['II']['order'], modules['III']['order']]),
 			'money': priority_match([modules['I']['money'], modules['II']['money'], modules['III']['money']]),
+			// create tokens
+			getNameTokens: getNameTokens,
```
Comment (Arachnid) [inline index.html:1717]: This will pass in the getNameTokens function as a template variable. Are you sure that's what you want?

Comment (someknave) [inline index.html:1717]: not certain if it is the best way to do this, I think that it is necessary for the function to be called inside params. if you can think of a better place for the function to be called or a better way for this to be handled in general let me know. but this does work.

Comment (Arachnid) [inline index.html:1717]: Where does it get called, other than in the line immediately below this? It's certainly not necessary for the following line; you can just delete this line entirely and it'll keep working.

Comment (someknave) [inline index.html:1717]: I was having a lot of issues getting this function call to work properly (I am not a js programmer) and this seemed to be best practice. that is the only place it is called so if this isn't necessary I'm happy for it to be removed.

**Pull request merged**